doc/external-signer.md

```diff
@@ -40,6 +40,8 @@ Create a wallet, this automatically imports the public keys:
 $ bitcoin-cli createwallet "hww" true true "" true true true
 ```
 
+Creating `N` distinctly named wallets with the same external signer, will create `N` identical wallets, with the same seed, account, and addresses - just named differently. New wallet creation will not automatically increment the `account` path. 
+
```

Comment:
Assuming this changed is correct, you'll need to appease the linter by removing the extra space.